src/SmartComponents/Recs/DetailsPathways.js

```diff
@@ -63,7 +64,50 @@ const PathwayDetails = () => {
     ...options,
     slug: pathwayName,
   });
-  const [activeTab, setActiveTab] = useState(0);
+  const { pathname } = useLocation();
+
+  const [activeTab, setActiveTab] = useState(
+    pathname.includes('/recommendations/pathways/systems/') ? 1 : 0
+  );
+
+  function waitForElm(selector) {
+    return new Promise((resolve) => {
+      if (document.querySelector(selector)) {
+        return resolve(document.querySelector(selector));
+      }
+
+      const observer = new MutationObserver(() => {
+        if (document.querySelector(selector)) {
+          resolve(document.querySelector(selector));
+          observer.disconnect();
+        }
+      });
+      observer.observe(document.body, {
+        childList: true,
+        subtree: true,
+      });
+    });
+  }
+
+  //This function runs only once, and only if they came from the systems affected link
+  var scrollDown = (function () {
+    var executed = false;
```

Comment:
Just out of my curiousity, can we use useEffect for executing only once?

Reply:
My mistake , I hadn't realized I had pushed up an older commit.